webpack.config.js

```diff
@@ -160,7 +161,10 @@ if ( calypsoEnv === 'desktop' ) {
 
 const jsLoader = {
 	test: /\.jsx?$/,
-	exclude: /node_modules/,
+	include: [
+		path.resolve( './client' ),
+		fs.realpathSync( './node_modules/notifications-panel' )
```

Comment:
Ah, I suppose we're building for two environments here, so this might not be appropriate. In general we use `peerDependencies` when we know something must be available, for example if we're authoring a React component library. For folks who want to use it, it's pretty safe to assume some version of React must be installed. ( In this example React would be added as a dev dependency in the project itself so development can occur ).

I'm thinking with the current approach that it may be possible to have differences pop up due to webpack configs drifting here in Calypso vs in the notifications client. The safest way to lock things down would still be to declare all dependencies, use the minified bundle in Calypso, and use `npm link` for faster development (though I haven't tried this with github links). The dep versions might drift, but things would still work. I'd assume we'd make a best effort to match up with what Calypso is using.